common/src/main/java/io/netty/util/ConstantPool.java

```diff
@@ -31,6 +31,7 @@
 
     private final ConcurrentMap<String, T> constants = PlatformDependent.newConcurrentHashMap();
 
+    @Deprecated
```

Comment:
Why you have this here ? Its private so you no need this. Just remove it if its not needed anymore.